src/cloud/howtos/debug.md

```diff
@@ -325,6 +325,16 @@ php -d xdebug.profiler_enable=On -d xdebug.max_nesting_level=9999 ...
 
 If you expect to use SSH and debug multiple times, you can put the export commands into a bash script in the `/tmp` directory to run them each time.
 
+## Debug CLI commands for Pro Staging and Production {#pro-debugcli}
```

Comment:
Instead of creating a separate section, I think it's better to include the information inline in the existing section.  However, the instructions in that section were a bit disjointed which makes it hard to integrate new content.  I suggest replacing the entire section, starting at line 301 with the content below the ----.  If the suggested revision introduced any technical errors or confusion, let's discuss.

---------
## Debug CLI commands {#debugcli}

When you debug CLI commands, you must configure environment variables to enable Xdebug. These variables are removed when the SSH session ends. If you expect to debug over multiple SSH sessions, add the export commands for the variable configuration to a bash script in the `/tmp` directory to run each time you start a session.

{:.procedure}
To enable Xdebug for debugging CLI commands:

1. SSH to the environment you want to debug.

1. Configure environment variables:

   ```bash
   export XDEBUG_CONFIG='PHPSTORM'
   ```

   ```bash
   export PHP_IDE_CONFIG="serverName=<name of the server that is configured in PHPSTORM>"
   ```

   Optionally, add Xdebug runtime options to the environment variable configuration.

   ```bash
   php -d xdebug.profiler_enable=On -d xdebug.max_nesting_level=9999 ...
   ```

1. Begin debugging.

   On Starter environments and Pro Integration environments, run the CLI command to debug, for example:

   ```bash
   php bin/magento cache:clean
   ```

   On Pro Staging and Production environments, you must specify the path to the Xdebug php configuration file when debugging CLI commands, for example:

   ```bash
   php -c /etc/platform/USERNAME/php.xdebug.ini bin/magento cache:clean
   ```